__tests__/app.test.js

```diff
@@ -104,3 +104,50 @@ test('200: Should return objects without body property', () => {
 			});
 		});
 });
+describe('PATCH /api/articles/:article_id', () => {
+	test('200: Should return article with updated vote count when passed a positive number', () => {
+		return request(app)
+			.patch('/api/articles/1')
+			.send({ inc_votes: 9 })
+			.expect(201)
+			.then(({ body }) => {
+				expect(body[0].votes).toBe(109);
+			});
+	});
+	test('200: Should return article with updated vote count when passed a negative number', () => {
+		return request(app)
+			.patch('/api/articles/1')
+			.send({ inc_votes: -150 })
+			.expect(201)
+			.then(({ body }) => {
+				expect(body[0].votes).toBe(-50);
+			});
+	});
+	test('404: valid but non-existent id', () => {
+		return request(app)
+			.patch('/api/articles/25')
+			.send({ inc_votes: -150 })
+			.expect(404)
+			.then(({ body }) => {
+				expect(body.msg).toBe('Invalid ID');
+			});
+	});
+	test('400: invalid id (NAN)', () => {
+		return request(app)
+			.patch('/api/articles/banana')
+			.send({ inc_votes: -150 })
+			.expect(400)
+			.then(({ body }) => {
+				expect(body.msg).toBe('Bad request');
+			});
+	});
+	test('400: invalid input passed (NAN)', () => {
+		return request(app)
+			.patch('/api/articles/1')
+			.send({ inc_votes: 'banana' })
+			.expect(400)
+			.then(({ body }) => {
+				expect(body.msg).toBe('Bad request');
+			});
+	});
```

Comment:
Please test for: 
-A non-existant ID (e.g. 999)

Reply:
> Please test for: -A non-existant ID (e.g. 999)

I have tested for that on line 126